python-src/webapp

```diff
@@ -0,0 +1 @@
+/home/lukas/uni/pintheworld/js-src/dist
```

Comment:
Should we remove this folder from git? Running `npm run build-dev` throws an exception the first time. Removing the this folder and running `npm run build-dev` fixes the problem.